pkg/types/kline.go

```diff
@@ -243,9 +243,10 @@ func (k *KLine) Color() string {
 }
 
 func (k *KLine) String() string {
-	return fmt.Sprintf("%s %s %s %s O: %.4f H: %.4f L: %.4f C: %.4f CHG: %.4f MAXCHG: %.4f V: %.4f QV: %.2f TBBV: %.2f",
+	return fmt.Sprintf("%s %s %s %s %s O: %.4f H: %.4f L: %.4f C: %.4f CHG: %.4f MAXCHG: %.4f V: %.4f QV: %.2f TBBV: %.2f",
 		k.Exchange.String(),
-		k.StartTime.Time().Format("2006-01-02 15:04"),
+		k.StartTime.Time().Format("2006-01-02 15:04:00.000"),
+		k.EndTime.Time().Format("2006-01-02 15:04:00.000"),
```

Comment:
The reason we didn’t print EndTime before was because it would’ve made the log message too long,

I think we can weigh it out a bit.

Reply:
fixed.
I only add ms part to the start time.

Reply:
thanks